test/geocel/vg/Vecgeom.test.cc

```diff
@@ -1034,7 +1120,8 @@ TEST_F(CmseTest, trace)
         static real_type const expected_distances[] = {12.495, 287.505, 530,
             920};
         EXPECT_VEC_SOFT_EQ(expected_distances, result.distances);
-        static real_type const expected_hw_safety[] = {1, 1, 242, 460};
+        static real_type const expected_hw_safety[] = {6.2475, 47.9499999999998,
+            242, 460};
```

Comment:
Guess we'll have to change the label on this one 😄 since it now agrees between the two